.github/workflows/test.yml

```diff
@@ -9,10 +9,10 @@ on:
     - master
 
 jobs:
-  build:
+  python-django:
     runs-on: ubuntu-latest
     strategy:
-      max-parallel: 5
```

Comment:
Please leave this in to make sure the number of GitHub actions runners shared throughout the Jazzband org are not overwhelmed. Thank you!